ecc/blocks/series-details-component/controller.js

```diff
@@ -34,8 +34,32 @@ export async function onPayloadUpdate(_component, _props) {
   // Do nothing
 }
 
-export async function onRespUpdate(_component, _props) {
-  // Do nothing
+export async function onRespUpdate(component, props) {
+  const { response } = props;
+  const cloudTypeEl = component.querySelector('#bu-select-input');
+  const targetCmsEl = component.querySelector('#targetcms-select-input');
+  const seriesNameEl = component.querySelector('#info-field-series-name');
+  const seriesDescriptionEl = component.querySelector('#info-field-series-description');
+
+  const hasResponse = Boolean(response);
```

Comment:
Since this whole function is just about prepopulate fields, I think we should just do a check after L38 and say if (!response) return;

Also, Instantiating a Boolean is a bit unnecessary to me and storing as variables is clean but might be a little unnecessary. I would just check their truthiness when we need to use them.